openedxscorm/scormxblock.py

```diff
@@ -24,23 +29,30 @@ def _(text):
 
 logger = logging.getLogger(__name__)
 
+# importing directly from settings.XBLOCK_SETTINGS doesn't work here... doesn't have vals from ENV TOKENS yet
+scorm_settings = settings.ENV_TOKENS['XBLOCK_SETTINGS']['ScormXBlock']
+SCORM_FILE_STORAGE_TYPE = scorm_settings.get("SCORM_FILE_STORAGE_TYPE", "django.core.files.storage.default_storage")
+SCORM_MEDIA_BASE_URL = scorm_settings.get("SCORM_MEDIA_BASE_URL", "/scorm")
```

Comment:
Can we use a single type of quotes (single or double)?

Reply:
Using double just for the consistency.

Reply:
Then could you replace other single quotes with doubles?

Reply:
done.